src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyForInitializerPadCheck.java

```diff
@@ -52,6 +52,20 @@
  * &lt;module name=&quot;EmptyForInitializerPad&quot;/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * for ( ; i &lt; 1; i++ ) { // violation semicolon is preceded with whitespace
+ * }
```

Comment:
Please add two more loops with no space in other places.

Reply:
done